packages/protocol/contracts/governance/Governance.sol

```diff
@@ -1052,4 +1053,50 @@ contract Governance is IGovernance, Ownable, Initializable, UsingLockedGold, Ree
     }
     return threshold;
   }
+
+  // call has been separated into its own function in order to take advantage
```

Comment:
Merge conflict? I believe this was recently moved into `Proposals.sol`

Reply:
yep, fixed